python/ray/tests/enable_tracing.py

```diff
@@ -0,0 +1,23 @@
+from ray.tests.test_tracing import spans_dir
```

Comment:
which version should we keep:
- tests/enable_tracing.py
- utils/enable_tracing.py

Reply:
so test/enable_tracing.py is slightly different because it uses the spans_dir that is the dir of the tmpfile from test_tracing.py , which is why i have both

Reply:
can you hard code spans_dir in the tests/enable_tracing.py and have test_tracing.py import from enable_tracing? In this way we can eliminate utils/enable_tracing.py